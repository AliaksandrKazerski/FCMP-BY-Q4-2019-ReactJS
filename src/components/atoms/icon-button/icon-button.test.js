import React from 'react';
import { shallow } from 'enzyme';

import IconButton from './index';

describe('IconButton', () => {
  it("IconButton should render correctly", () => {
  const myComponent = shallow(<IconButton/>);
  
  expect(myComponent).toMatchSnapshot()
  });
});