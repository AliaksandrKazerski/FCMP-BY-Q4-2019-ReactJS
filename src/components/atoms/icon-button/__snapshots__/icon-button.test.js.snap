// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary ErrorBoundary should render correctly 1`] = `ShallowWrapper {}`;

exports[`IconButton IconButton should render correctly 1`] = `ShallowWrapper {}`;
